lib/board.rb

```diff
@@ -1,11 +1,29 @@
 require 'pry'
+require './lib/cell'
 
 class Board
+	attr_reader :contents
+
 	def initialize
-  	@contents = Array.new(7) { Array.new(6) { |i| i = '.' } }
-  end
+		@contents = Array.new(6) { Array.new(7) }
+	end
+
+	def populate_board
+		for row in 1..6 do
+			for column in 1..7 do
+				@contents[row - 1][column - 1] = Cell.new(row, column, 'x')
```

Comment:
the x should be changed to a '.' ---to follow the expectation for an empty board.